### ISSUE TITLE
Browser extension - hover cards don't show on GitHub Enterprise

### ISSUE THREAD

**Issue by inazarenko** [sourcegraph/sourcegraph#13492]
- **Sourcegraph version:** 

Version: 3.19.1

- **Platform information:** 

Browser: Chrome, Version 84.0.4147.135 (Official Build) (64-bit), Mac OS (also tested on Linux)

#### Steps to reproduce:

1. Configured extension to point at local SG instance at https://sg.corp.(redacted intranet domain). Extension says "connected"
2. Enabled extension on our local GitHub Enterprise domain (Server 2.19.20).
3. Verified extension buttons show up ("view on Sourcegraph") and work (open tabs to expected SG instance)

#### Expected behavior:

Mouse-hover over identifiers on GitHub Enterprise should show SG's hover cards

#### Actual behavior:

No hover cards and no activity in dev console from the extension.

<img width="429" alt="Screen Shot 2020-08-31 at 9 56 17 AM" src="https://user-images.githubusercontent.com/12076208/91757170-c4e77380-eb82-11ea-8313-d414149e3fe8.png">

**Comment by christinelovett** [comment]
https://app.hubspot.com/contacts/2762526/company/1021335057

**Comment by tistru** [comment]
Update from customer:

Tried reinstalling Chrome add-on; no change in behavior.

I verified that the extension does work on github.com when configured to talk to sourcegraph.com.

Installed Firefox, same outcome: no hover cards on our GitHub, console log below. I do get "view in SG" buttons, just not the hovers.

Cookie “has_recent_activity” will be soon rejected because it has the “SameSite” attribute set to “None” or an invalid value, without the “secure” attribute. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite iter.h
Sourcegraph browser extension is running inject.bundle.js:38544
Detected code host: github inject.bundle.js:38544
Browser extension is enabled inject.bundle.js:38544
